containers/NavigatorRedux.js

```diff
@@ -40,17 +40,19 @@ renderNavigator.propTypes = {
 };
 
 renderNavigator.defaultProps = {
-  renderer: (route) => (
-    <View style={styles.container}>
-      {route}
-    </View>
-  ),
+  renderer: (route) => {
+    return (
+      <View style={styles.container}>
+        {route}
+      </View>
+    );
+  },
 };
 
 module.exports = connect(
   (state) => ({
-    stack: state.__nav.stack,
-    index: state.__nav.index,
+    stack: state.__navRedux.stack,
```

Comment:
You removed it in a previous commit because naming conflicts.
I just renamed to continue having a nested object.